data/trivia/entertainment.txt

```diff
@@ -530,7 +530,7 @@ How old was Shirley Temple when she made her last film?`21
 Who is Declan Patrick McManus better known as?`Elvis
 Where did Mighty Mouse get his superpowers?`Supermarket
 Who broke Batman?`Bane
-On what LP Cover can we read the words 'Welcome Rolling Stones'?`Sergeant Pepper�s Lonely Hearts Club Band
+On what LP Cover can we read the words 'Welcome Rolling Stones'?`Sergeant Pepper´s Lonely Hearts Club Band

```

Comment:
This `´` character should be an apostrophe `'`

Reply:
good catch @_@